### PR TITLE
add ngsw-manifest and gh-pages deployment

### PR DESCRIPTION
**Pull request by wassertim**
we need this for presenting the PWA and offline mode

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,7 +1,6 @@
 # See http://help.github.com/ignore-files/ for more about ignoring files.
 
 # compiled output
-/dist
```
Comment (d3ns0n) [inline .gitignore]: Why did you remove this from .gitignore? Do we need to put `dist` under version control?

**Comment by wassertim** [comment]
Yes, it has to be there if we want to use github pages

Tim

**Comment by wassertim** [comment]
BTW: https://wassertim.github.io/js-days-recap/ check it out :)

Tim

**Comment by d3ns0n** [comment]
Yeah looks cool :) but I am not sure if I want to add `dist` to git because it will change every time you build the project. So you'd have to either ignore it manually or commit it every time you change something. We should talk it through tomorrow. 

Maybe we could use travis to build and commit to another repository. Just an idea.

**Comment by wassertim** [comment]
Good idea. There is some info about that https://docs.travis-ci.com/user/deployment/pages/. I'm not sure if I would have time for this, I'll have to do slides tomorrow :(

File: README.md [hunk 2]
```diff
@@ -1,4 +1,5 @@
 # TODO App with angular 4 universal
+![Build Status](https://img.shields.io/badge/js--days--recap-1.0.0-brightgreen.svg)
```
Comment (wassertim) [inline README.md]: haha :)

**Comment by wassertim** [comment]
great work @d3ns0n
